metaflow/plugins/aws/batch/batch.py

```diff
@@ -220,7 +195,10 @@ def launch_job(
         self,
         step_name,
         step_cli,
-        task_spec,
+        flow_name,
```

Comment:
nit: I like replacing task_spec with explicit components but if we do that, it may be clearer to put them in order (here step_name is before and then we see flow_name, run_id and task_id (step_name should ideally be in between)).

Reply:
fixed